src/components/add-class/AddClass.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import './addClass.css';
+import { createClass } from '../../redux/addClass/addClassSlice';
+
+const ClassForm = () => {
+  const dispatch = useDispatch();
+  const [subject, setSubject] = useState('');
+  const [description, setDescription] = useState('');
+  const [price, setPrice] = useState('');
+  const [duration, setDuration] = useState('');
+  const [image, setImage] = useState('');
+  const [tutorName, setTutorName] = useState('');
+  const [submitted, setSubmitted] = useState(false);
+  const [error, setError] = useState(false);
```

Comment:
[OPTIONAL] Rather than managing each form field with its own state, we can use one state object for the entire form. This reduces the number of useState calls and makes it easier to manage the form state as a whole.